JASP-Engine/JASP/R/descriptives.R

```diff
@@ -117,6 +117,23 @@ Descriptives <- function(jaspResults, dataset, options) {
       }
     }
   }
+  # Interval plots
+  if (options$descriptivesIntervalPlot) {
+    if(is.null(jaspResults[["IntervalPlots"]])) {
+      jaspResults[["IntervalPlots"]] <- createJaspContainer(gettext("Intervalplots"))
+      jaspResults[["IntervalPlots"]]$dependOn(c("IntervalPlots", "splitby"))
+      jaspResults[["IntervalPlots"]]$position <- 11
+    }
+
+    IntervalPlots <- jaspResults[["IntervalPlots"]]
```

Comment:
I know it's very pedantic, but just to be consistent with camelcase could you make the first letter lowercase?
```suggestion
    intervalPlots <- jaspResults[["intervalPlots"]]
```
see also https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/r-style-guide.md